homeassistant/components/intellifire/sensor.py

```diff
@@ -85,6 +86,42 @@ def _time_remaining_to_timestamp(data: IntellifirePollData) -> datetime | None:
         device_class=SensorDeviceClass.TIMESTAMP,
         value_fn=_time_remaining_to_timestamp,
     ),
+    IntellifireSensorEntityDescription(
+        key="downtime",
+        name="Downtime",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.downtime,
+    ),
+    IntellifireSensorEntityDescription(
+        key="uptime",
+        name="Uptime",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.uptime,
```

Comment:
We do not allow relative times to be stored in the state machine. Instead, make it a device class TIMESTAMP and store all the time it went online.

Reply:
Internally I modified this to: 

```python
    IntellifireSensorEntityDescription(
        key="uptime",
        name="Uptime",
        entity_category=EntityCategory.DIAGNOSTIC,
        device_class=SensorDeviceClass.TIMESTAMP,
        value_fn=lambda data: utcnow() + timedelta(seconds=data.uptime)
    ),
```

And it looks like:

<img width="253" alt="image" src="https://user-images.githubusercontent.com/6491743/154677731-b180a7d0-aeb4-494b-b8a5-d14fb42eff71.png">

    
Is there a way to get rid of the **In**?

I'm using:

`from homeassistant.util.dt import utcnow`

 as my import

Reply:
I think I can work with `8 minutes ago` if there isn't a `for` option.

Looking over the other integrations except for UPNP:

<img width="394" alt="image" src="https://user-images.githubusercontent.com/6491743/154682469-b03e7cf0-cf90-49f0-a25d-2e2eca58c194.png">

which seems to use raw seconds - seems most other integrations are going with:

```
utcnow() - timedelta(seconds=data.uptime)
```